notebooks/finetune/Flan-T5-3B/RosaQA.ipynb

```diff
@@ -0,0 +1,7957 @@
+{
```

Comment:
<p>Where is this dataset?</p><p> </p><p>Can we maybe show a few rows from it to illustrate how it looks like?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>+1</p>